drivers/can/Kconfig

```diff
@@ -273,6 +273,15 @@ config CAN_KVASER
 
 if CAN_KVASER
 
+config CAN_KVASER_IRQ_PASSES
+	int "Kvaser PCI interrupt passes"
+	default 16
```

Comment:
Please include a range here, if the user pass a zeroed or negative value the "for" loop will run "forever" (until overflow)